server/Arcadia.Assistant.CSP/Vacations/ArcadiaVacationRegistry.cs

```diff
@@ -66,12 +81,107 @@ protected override void OnReceive(object message)
 
                     break;
 
+                case CalendarEventCreated msg when msg.Event.Type == CalendarEventTypes.Vacation:
```

Comment:
case CalendarEventCreaten when msg.Event.Type != CalendarEventTypes.Vaction: break? At the very beginning.

Reply:
As you wish, but it doesn't look much prettier.

Reply:
Done.